### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by ammarAlaliS**
Description:
This pull request aims to enhance our team's knowledge and skills in front-end web development by focusing on the following learning objectives:

Utilizing Medium-Fidelity Wireframes: Learn how to translate design ideas into UI elements using medium-fidelity wireframes, promoting effective collaboration between designers and developers.

Following Written Requirements: Practice implementing user stories and written requirements to ensure accurate development and alignment with project goals.

Applying Semantic HTML Tags: Understand the importance of semantic HTML tags for accessibility, SEO, and maintaining a well-structured website.

Implementing Best Practices in HTML: Apply industry-standard HTML best practices to produce clean, maintainable, and efficient code.

Correctly Using CSS Selectors: Gain proficiency in using CSS selectors to target and style specific HTML elements effectively.

Understanding CSS Box Model: Comprehend the CSS box model and its impact on element sizing and spacing to create consistent and visually appealing layouts.

Utilizing Flexbox for Layouts: Use Flexbox to efficiently arrange and align elements within web pages, allowing for responsive and dynamic layouts.

Enhancing UI with Images and Backgrounds: Explore techniques to incorporate images and backgrounds effectively, improving the visual appeal of websites.

Creating Responsive UIs with Media Queries: Demonstrate the ability to build adaptive user interfaces that respond appropriately to various screen sizes using media queries.

Deploying Web Pages with GitHub Pages: Learn how to deploy web pages using GitHub Pages, making projects accessible to a broader audience.

Adhering to JavaScript Best Practices: Apply industry-standard JavaScript best practices and adhere to language style guides, ensuring clean and maintainable code.

Manipulating DOM Elements with JavaScript: Gain proficiency in using JavaScript to dynamically modify and interact with the Document Object Model (DOM).

Handling JavaScript Events: Understand event handling in JavaScript and use it to create interactive and responsive user experiences.

Utilizing Objects for Data Storage: Learn how to use objects to store and access data efficiently, promoting code organization and reusability.

By accomplishing these learning objectives, we aim to strengthen our team's front-end development skills, improve the quality of our codebase, and enhance the overall user experience in our projects.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                    <i class="fa-solid fa-xmark"></i>
+                </div>
+                <ul class="menuLinks">
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="./index.html#courses">Courses</a></li>
+                    <li><a href="./index.html#Sponsor">Sponsor</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
```
Comment (ridwanediallo) [inline about.html:39]: - [ ] You did a fantastic job on the navigation 👏. To ensure that you stay true to the original design, I recommend adding the top navigation bar with the black background color. Please also include the orange borders around the **Campaing*** link. :ok_hand:  🤓 

### Your design  :point_down: 
Screenshot from 2023-08-02 23-45-00


### The original design :point_down: 
Screenshot from 2023-08-02 23-44-30

File: index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                    <i class="fa-solid fa-xmark"></i>
+                </div>
+                <ul class="menuLinks">
+                    <li><a  href="./about.html">About</a></li>
+                    <li><a  href="#courses">Courses</a></li>
+                    <li><a href="#Sponsor">Sponsor</a></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
```
Comment (ridwanediallo) [inline index.html:41]: - [ ] You did a fantastic job on the navigation 👏. To ensure that you stay true to the original design, I recommend adding the top navigation bar with the black background color. Please also include the orange borders around the **Campaing*** link. :ok_hand:  🤓 
### Your design  :point_down: 
Screenshot from 2023-08-03 00-08-52



### The original design :point_down: 
Screenshot from 2023-08-02 23-44-30

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @ammarAlaliS , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️

- [ ] Please ensure that the description section in the readme file contains a full description of the project rather than just the name of the project to make your readme file more descriptive.  :ok_hand: 😊

#### Description section :point_down: 
Screenshot from 2023-08-03 00-40-58

- [ ] Last but not least kindly don't leave the contributing section emplty.
#### Contibuting section :point_down: 
Screenshot from 2023-08-03 00-41-08



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @ammarAlaliS,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  No Linters errors 🔥
- [ ]  Great job working on the functionality 💯
- [ ]  Followed GitHub Flow ⭐🚀
- [ ]  Descrtipive PR⭐
- [ ] Great design💯
- [ ]  Kudos for working on the requested changes💯
- [ ] Professional README 🎉💪🏻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**